bindings/c/test/unit/unit_tests.cpp

```diff
@@ -2047,6 +2047,7 @@ TEST_CASE("fdb_database_force_recovery_with_data_loss") {
 }
 
 std::string random_hex_string(size_t length) {
+	srand(time(0)); // set random seed
```

Comment:
We should probably be using modern c++ random api's: http://www.cplusplus.com/reference/random/

Reply:
Yeah, sorry for this. I just realized I did not initialize the seed thus it is not random every time.
Let me fix this quickly.

Reply:
Done. Thanks.